src/inputs/dropdownButton.jsx

```diff
@@ -6,16 +6,29 @@ import Button from './button';
 import DropdownMenu from './dropdownMenu';
 import DropdownMenuOption from './dropdownMenuOption';
 import Icon from '../dataDisplay/icon';
+import makeStyles from '../styles/makeStyles';
 
 const propTypes = {
     ...buttonPropTypes,
+    classes: PropTypes.shape({
+        root: PropTypes.string,
+    }),
+    /**
+     * The icon that sits to the left of the label.
```

Comment:
Should this be 'The _size of the_ icon that sits to the left of the label.'?

Reply:
Fixed.

Reply:
Yep, update in 170fcf6 looks good.  Resolving.